CHANGELOG.md

```diff
@@ -1,5 +1,73 @@
 # CHANGELOG
 
+## Release 2.4.4 (in development)
+
+### Breaking changes
+- Due to a critical vulnerability with CVSS score of 10.0 that was discovered
+affecting all `Salt Master` versions inferior to `3000.1`, this release ships
+with a hot fix and of course a `Salt Master` version updated to `3000.3`.
+Users, especially those who expose the `Salt Master` to the internet must
+therefore upgrade immediately.
+
+- The official `python-kubernetes client v11` has recently been released. The new
+client comes with new API changes and a couple of old APIs deprecated.
+MetalK8s has been updated to fully support this new client API version.
```

Comment:
Not sure this is a breaking change either: the v11 client is matching K8s 1.15, which is what we deploy with MetalK8s 2.4. The change should be transparent for 2.4 users, maybe not worth stating here (although it's a bugfix, because previous versions of 2.4.x used the old v10 client, which didn't include new APIs from K8s 1.15).